common/utils/src/main/resources/error/error-classes.json

```diff
@@ -5142,7 +5143,7 @@
   },
   "_LEGACY_ERROR_TEMP_1183" : {
     "message" : [
-      "Cannot use interval type in the table schema."
+      "Cannot use \"INTERVAL\" type in the table schema."
```

Comment:
BTW, it is unrelated changes, it seems.

Reply:
If it is not related, please, revert this and the modified tests.

Reply:
it isn't, there is a test `SQLInsertTestSuite.scala` checking for both cases where interval is used in a schema or in a partition column, so when I updated the error class for partitioning to use sql type it broke the test for checking for interval type in the schema

Reply:
ok. I see.